src/cpp/client/ProxyClient.cpp

```diff
@@ -37,6 +37,8 @@ ProxyClient::ProxyClient(
     : representation_(representation)
     , objects_()
     , session_(SessionInfo{representation.client_key(), representation.session_id(), representation.mtu()})
+    , state_{State::alive}
```

Comment:
Where are we setting the state to `State::dead`?.
What are we doing with the "dead" Clients?.

Reply:
We check the `ProxyClient` state at `Processor::check_heartbeats`.
In case the `ProxyClient` is dead, we do not send **HEARTBEAT**s.